ethstore/src/accounts_dir/disk.rs

```diff
@@ -117,16 +119,30 @@ pub struct DiskDirectory<T> where T: KeyFileManager {
 }
 
 /// Keys file manager for root keys directory
-pub struct DiskKeyFileManager;
+pub struct DiskKeyFileManager {
+	password: Option<Password>,
+}
+
+impl DiskKeyFileManager {
+	/// Create new disk key manager w/o password
+	pub fn new() -> Self {
```

Comment:
You could derive this impl instead with a default implementation which I prefer in this context!